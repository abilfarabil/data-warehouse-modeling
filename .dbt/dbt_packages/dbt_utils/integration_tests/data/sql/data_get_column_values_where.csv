version https://git-lfs.github.com/spec/v1
oid sha256:884fb4dcab9cccaab9b02b188a4c61966300f3e5efecf9023bc8d39f51138a6a
size 108
