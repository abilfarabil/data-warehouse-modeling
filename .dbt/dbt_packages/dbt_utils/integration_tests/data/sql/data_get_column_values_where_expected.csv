version https://git-lfs.github.com/spec/v1
oid sha256:f6796de2efb9f5d95c88ad088eddd5e8d9bbe815be1739abe9ae28b110bafa52
size 17
